semver.md

```diff
@@ -319,6 +319,22 @@ Abbreviating "version" as "v" is often seen with version control. Example:
 `git tag v1.2.3 -m "Release version 1.2.3"`, in which case "v1.2.3" is a tag
 name and the semantic version is "1.2.3".
 
+### What does SemVer consider a package?
+
+The meaning of package in this specification describes a deployment unit.
+It could be built from a single or multiple source artifacts. But it can
+also be an assembly of multiple binary artifacts.
+
```

Comment:
No need for one sentence paragraphs here.  Just connect the above to the one below and then the "however" in that lower sentence will not be cognitively burdensome.

Reply:
Yes correct, was doing too much Asciidoc lately. And I will remove ‚the meaning‘

Reply:
"..." means whatever came before or whatever comes after. No need to insert them in the text.